rosplan_planning_system/launch/includes/planner_interface.launch

```diff
@@ -6,9 +6,9 @@
 	<arg name="use_problem_topic"    default="true" />
 	<arg name="problem_topic"        default="/rosplan_problem_interface/problem_instance" />
 	<arg name="planner_topic"        default="planner_output" />
-	<arg name="domain_path"          default="$(find rosplan_planning_system)/test/turtlebot_demo/domain_turtlebot_demo.pddl" />
-	<arg name="problem_path"         default="$(find rosplan_planning_system)/test/turtlebot_demo/problem.pddl" />
-	<arg name="data_path"            default="$(find rosplan_planning_system)/test/turtlebot_demo/" />
+	<arg name="domain_path"          default="$(find rosplan_planning_system)/test/pddl/turtlebot/domain.pddl" />
```

Comment:
Does this belong here? not sure if the rosplan_system launches should have hardcoded domains.  Same applies to the other launches below.

Reply:
Short story: yes it does. (see motivation below)

Earlier it pointed to rosplan demos:
```
<arg name="domain_path" default="$(find rosplan_demos)/common/domain_turtlebot_demo.pddl" />
```
When rosplan_demos were de attached from main implementation the idea was to make a test domain, which could be reused as well for unit test purposes while keeping this repository self-contained (fully independent of rosplan_demos). Later on, the unit tests grew to include an amazon and a small roomba domain (contributed by @martinkoling), so this change here is an attempt to organize the different domains in folders.

Reply:
Agree and understood, but still not sure that the planner_interface should have hardcoded paths, the test should, but don't know if the interface one. @m312z what are your thoughts on this? I don't have a clear answer either.

Reply:
The unit test launch file does not need a harcoded path, but reuses the example launch file (taking advantage of being a parameterized one).

We could do something like:

```<arg name="domain_path" default="foo" />```

and tests will still be able to work, however I think is more clear to the user to provide a full self contained example of how to use the launch file.

Reply:
I agree with Oscar on this one. The launch files contain default values that work (pointing to some example domain) and keep the repo self-contained. I think it is more helpful when learning how things work to see exactly where the domain file is referenced.